org.kde.kasts.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "org.kde.kasts",
+    "rename-icon": "kasts",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kasts",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--own-name=org.mpris.MediaPlayer2.kasts"
+    ],
+    "modules": [
+        {
+            "name": "kasts",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [ "-DBUILD_TESTING=OFF", "-DCMAKE_BUILD_TYPE=RelWithDebInfo" ],
```

Comment:
As far as i can tell flathub doesn't usually do RelWithDebInfo builds

Also, we don't have tests, so disabling them is not really necessary